src/digits_sqd.py

```diff
@@ -0,0 +1,11 @@
+def digits_squared(num):
+    digits = str(num)
+    sol = ""
```

Comment:
Try to avoid using abbreviations. This may be obvious to you, but sol is ambiguous. result would be better, or solution.